app/assets/stylesheets/partials/_cafe.scss

```diff
@@ -0,0 +1,30 @@
+.cafe {
+  &.working {
+    color: $fsek-orange;
+    display: block;
+    font-weight: bold;
+    margin: 20px 0 20px 0;
+  }
+
+  tr.with_worker {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector `with_worker` should be written in lowercase with hyphens